modules/virt-importing-vm-prerequisites.adoc

```diff
@@ -30,11 +30,11 @@ Source environment prerequisites::
 ** The VM must not be configured for USB devices.
 ** The watchdog model must not be `diag288`.
 
-Target environment prerequisites::
+== Target environment prerequisites
 
-* The {product-title} storage class must be NFS.
+* The storage class must be NFS.
+* The local and shared persistent storage must support VM import.
```

Comment:
What this refers to?

Reply:
These lines refer to RHV. You should check the preview build rather than the actual files because there is a lot of tagging.

Reply:
Would you please add a direct link to the preview ?
I could not find this table clicking on the above preview (and searching for it there).

Reply:
Preview links:

https://cnv-vm-import-storage-matrix--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/importing_vms/virt-importing-rhv-vm.html#virt-features-for-storage-matrix_virt-importing-rhv-vm

https://cnv-vm-import-storage-matrix--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/importing_vms/virt-importing-vmware-vm.html#virt-features-for-storage-matrix_virt-importing-vmware-vm